docs/pages/indexer.mdx

```diff
@@ -388,11 +388,22 @@ If you are indexing a different `World`, change the world address and use the na
 1. Specify the world schema is the default.
 
    ```sql copy
-   set search_path to "0x5FbDB2315678afecb367f032d93F642f64180aa3__";
+   SET search_path TO "0x5FbDB2315678afecb367f032d93F642f64180aa3__";
    ```
 
 1. Get the value of the counter.
 
    ```sql copy
-   select * from "Counter";
+   SELECT * FROM "Counter";
    ```
+
+### Deleting the PostgreSQL data
+
+PostgreSQL is a persistent data storage.
+This means that if you restart the blockchain without manually deleting the data, your indexer will use old data and provide incorrect results.
+
+To avoid this, enter `psql` and run this command:
+
+```sql copy
+DROP OWNED BY <your user name>;
```

Comment:
what happens when people execute within docker? docker has different usernames I think.
should we drop the DB instead?

Reply:
The docker directions cover this.